components/net/http_loader.rs

```diff
@@ -1576,16 +1586,18 @@ async fn http_network_or_cache_fetch(
     {
         // TODO: Step 14.1 Spec says requires testing on multiple WWW-Authenticate headers
 
+        let request = &mut fetch_params.request;
+
         // Step 14.2 If request’s body is non-null, then:
-        if http_request.body.is_some() {
+        if request.body.is_some() {
```

Comment:
change this and the following to usages of `http_request` because the spec doesn't use `http_request` in steps 14 and 15